test/k8sT/Policies.go

```diff
@@ -1336,8 +1337,8 @@ EOF`, k, v)
 			demoPath        string
 			demoManifestNS1 string
 			demoManifestNS2 string
-			firstNS         = "first"
-			secondNS        = "second"
```

Comment:
We should be able to call `helpers.GenerateNamespaceForTest` from here, no? Or are you worried that the timestamp will be wrong? (Would it matter, though? it just needs to be different than another test running and ginkgo will run the tests one-by-one no matter what)

Reply:
Running it here causes a segfault because `ginkgo.CurrentGinkgoTestDescription()` returns `nil` when run in `Context`.